examples/player/simid_player.html

```diff
@@ -78,9 +79,18 @@
         <button onclick="switchCreative('survey.html')">survey</button>
         <button onclick="switchCreative('selector.html')">selector</button>
         <button onclick="switchCreative('extender.html')">extender</button>
+        <button onclick="switchCreative('nonlinear_overlay.html')">nonlinear_overlay</button>
```

Comment:
Add to this file something that sets defaults of all the UI when we select this creative, including the linear/non linear radio button.

Reply:
The input for the dimensions is added in the other PR so I'll add it there for the full functionality